orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/TaskController.groovy

```diff
@@ -583,38 +589,51 @@ class TaskController {
   @PreAuthorize("hasPermission(#application, 'APPLICATION', 'READ')")
   @RequestMapping(value = "/applications/{application}/pipelines", method = RequestMethod.GET)
   List<PipelineExecution> getPipelinesForApplication(@PathVariable String application,
-                                                         @RequestParam(value = "limit", defaultValue = "5")
-                                               int limit,
-                                                         @RequestParam(value = "statuses", required = false)
-                                               String statuses,
-                                                         @RequestParam(value = "expand", defaultValue = "true") Boolean expand) {
+                                                     @RequestParam(value = "limit", defaultValue = "5") int limit,
+                                                     @RequestParam(value = "statuses", required = false) String statuses,
+                                                     @RequestParam(value = "expand", defaultValue = "true") Boolean expand) {
     if (!front50Service) {
       throw new UnsupportedOperationException("Cannot lookup pipelines, front50 has not been enabled. Fix this by setting front50.enabled: true")
     }
 
     if (!limit) {
       return []
     }
-
     statuses = statuses ?: ExecutionStatus.values()*.toString().join(",")
     def executionCriteria = new ExecutionCriteria(
       pageSize: limit,
       statuses: (statuses.split(",") as Collection)
     )
 
+    // get all relevant pipeline and strategy configs from front50
     def pipelineConfigIds = front50Service.getPipelines(application, false)*.id as List<String>
+    log.info("received ${pipelineConfigIds.size()} pipelines for application: $application from front50")
```

Comment:
Same nit on log level - this seems more useful as a debug level log.

Reply:
changed to debug

Reply:
Agree with @jasonmcintosh that especially in the TaskController there are a lot of log.info that will fill out the console :) would prefer to keep in info only what is absolutely necessary since TaskController is heavily called